### PR TITLE
NativeAssemblyReference Include "MklProxyNative" in Samples project.

### LINKED ISSUE #3818: NativeAssemblyReference Include "MklProxyNative" in Samples project.
Need to the above dependency to have some timer series samples work that use MKL

### PR DESCRIPTION
**Pull request by codemzs**
fixes #3818

### CODE REVIEW

File: src/Microsoft.ML.TimeSeries/Microsoft.ML.TimeSeries.csproj [hunk 1]
```diff
@@ -5,6 +5,10 @@
     <IncludeInPackage>Microsoft.ML.TimeSeries</IncludeInPackage>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML.Mkl.Redist" Version="1.0.0" />
```
Comment (eerhardt) [inline src/Microsoft.ML.TimeSeries/Microsoft.ML.TimeSeries.csproj]: Isn't this needed in `pkg\Microsoft.ML.TimeSeries\Microsoft.ML.TimeSeries.nupkgproj` instead?

Comment (eerhardt) [reply src/Microsoft.ML.TimeSeries/Microsoft.ML.TimeSeries.csproj]: Instead of `PackageReference`, shouldn't it be a `ProjectReference`?

Comment (shmoradims) [inline src/Microsoft.ML.TimeSeries/Microsoft.ML.TimeSeries.csproj]: how was timeseries working so far without adding this dependency?

Comment (codemzs) [reply src/Microsoft.ML.TimeSeries/Microsoft.ML.TimeSeries.csproj]: It works with the nuget, not in samples repo for some time series samples that require mkl. I have fixed it now.,

**Pull request merged**